src/server/views/help/cookies.html

```diff
@@ -34,6 +39,33 @@ <h2 class="govuk-heading-m">Essential cookies</h2>
         ]
       ]
     }) }}
+
+    <h2 class="govuk-heading-m">Analytics cookies</h2>
+    <p class="govuk-body">We use Google Analytics cookies to collect information about how you use this form.</p>
```

Comment:
Just a suggestion, then we can talk separately about what we collect further down

```suggestion
    <p class="govuk-body">We use Google Analytics software to understand how people use our forms. We do this to help make sure the site is meeting the needs of its users and to help us make improvements.</p>
```